tests/common/mod.rs

```diff
@@ -6,6 +6,7 @@ use std::{
 
 use alloy_primitives::{uint, Address, Bloom, Bytes, B256, U256};
 use alloy_rpc_types::{Block, BlockTransactions, Header, Signature};
+use flate2::bufread;
```

Comment:
I just realized putting `bufread::GzDecoder` here would be cleaner & more symmetric.